lms/djangoapps/verify_student/models.py

```diff
@@ -1108,3 +1109,23 @@ def deadline_for_course(cls, course_key):
             return deadline.deadline
         except cls.DoesNotExist:
             return None
+
+
+class SSOReverificationConfig(ConfigurationModel):
+    """
+        SSOVerificationConfig
+    """
+
+    class Meta(object):
+        app_label = 'verify_student'
+        verbose_name = 'sso verify student argument'
+
+        arguments = models.TextField(
+            blank=True,
+            help_text='Useful for manually running a Jenkins job. Specify like "SSOVerification id,  \
```

Comment:
I believe help text should be "Add comma separated verification ids like `verification id 1, verification id 2,..."